R-Tutorial/scripts/Installing_R.md

```diff
@@ -0,0 +1,17 @@
+# The website for downloading the R binary code is:
+
+[R-CRAN-PROJECT-DOWNLOAD](https://cloud.r-project.org/)
+
+# Once downloaded, the following steps are followed :
+
+![Install_R_img1](../images/install_R_img1.jpg)
```

Comment:
I saw this [here](https://github.com/TheAlgorithms/R/blob/0b654a492818586c8eaf1929d369e8f481b7c1fc/R-Tutorial/scripts/Installing_R.md), the image just contains text on a blank background..why not write that line down instead? (don't forget to remove the image)